pkgs/tools/video/svt-av1/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, yasm }:
+
+stdenv.mkDerivation rec {
+  pname = "svt-av1";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "AOMediaCodec";
+    repo = "SVT-AV1";
+    rev = "v${version}";
+    sha256 = "1gfxdmdql090p7c8ln1z344g467l41p45287lmwy590hv8gw6bfg";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ yasm ];
```

Comment:
`yasm` should be in `nativeBuildInputs`, Their build system does support `nasm` and I would strongly recommend it over yasm.

Reply:
Done.